tests/foreman/ui/test_organization.py

```diff
@@ -25,15 +25,13 @@
 from robottelo.constants import INSTALL_MEDIUM_URL
 from robottelo.constants import LIBVIRT_RESOURCE_URL
 from robottelo.logging import logger
-from robottelo.manifests import original_manifest
-from robottelo.manifests import upload_manifest_locked
 
 CUSTOM_REPO_ERRATA_ID = settings.repos.yum_0.errata[0]
 
 
 @pytest.fixture(scope='module')
 def module_repos_col(module_org, module_lce, module_target_sat, request):
-    upload_manifest_locked(org_id=module_org.id)
+    module_target_sat.upload_manifest(org_id=module_org.id)
```

Comment:
Same issue as above, this will need either a manifest fixture or `function_entitlement_manifest_organization`.

Reply:
I made the changes in upload_manifest method itself and it's working. Will that be good?

Reply:
Yes, that' s fine for now.